### ISSUE TITLE
remove non unicode bytes from HISTORY.rst and README.md

### ISSUE THREAD

**Issue by anneverheijen** [openearth/dfm_tools#93]
* dfm_tools version: 0.7.23
* Python version: 3.8
* Operating System: Win 10

### Description

Install toolbox from conda .yml file

### What I Did

```
did: conda env update -f newenvironment.yml
with following lines in yml that give error
dependencies:
  - pip:
    - git+https://github.com/openearth/dfm_tools.git@6ac91e323ad9228cd10903a3e6af4ac15ad72b20

error:

File "\dfm_tools\setup.py", line 8, in <module>
        readme = readme_file.read()
      File "\lib\codecs.py", line 322, in decode
        (result, consumed) = self._buffer_decode(data, self.errors, final)
    UnicodeDecodeError: 'utf-8' codec can't decode byte 0xa0 in position 3617: invalid start byte

similar error will be found later in the history file
```

### What I expected:
seamless installation ;)

**Comment by anneverheijen** [comment]
@veenstrajelmer  Is it correct I'm not able to make any changes? I would like to change the HISTORY.rst and README.md file to the files attached

[HISTORY.rst.txt](https://github.com/openearth/dfm_tools/files/4912435/HISTORY.rst.txt)
[README.md.txt](https://github.com/openearth/dfm_tools/files/4912436/README.md.txt)

**Comment by veenstrajelmer** [comment]
@anneverheijen, thanks for your valuable feedback. The issue should be resolved in #94 (dfm_tools 0.7.24). Let me know if your issue was not resolved
